corehq/apps/registration/models.py

```diff
@@ -184,3 +185,4 @@ def complete_workflow(self, edition):
         self.completed_on = datetime.datetime.now()
         self.subscribed_edition = edition
         self.save()
+        self.get_in_progress_for_domain.reset_cache(self.domain)
```

Comment:
If `get_in_progress_for_domain` is cached (regardless of memoization vs quickcache), shouldn't that also be cleared when a new `SelfSignupWorkflow` is created?

Reply:
Not in any current or planned implementation, since the workflow object is only created at the same time the domain is, here (part of `request_new_domain`): https://github.com/dimagi/commcare-hq/blob/41e0db5b51c5c3582e2d6b522de115b569e62ac3/corehq/apps/registration/utils.py#L169-L170